docs/content/preview/contribute/docs/docs-style.md

```diff
@@ -56,7 +58,54 @@ ysqlsh> CREATE TABLE $$__banking__$$;
 
 #### YSQL and YCQL code blocks
 
-Tag YSQL code blocks as `sql`, and YCQL code blocks as `cql`. The source highlighting differs slightly between the two.
+Tag YSQL code blocks as `plpgsql`, and YCQL code blocks as `cql`. The source highlighting differs slightly between the two.
+
+### Tables
```

Comment:
Content more accurately goes on page https://docs.yugabyte.com/preview/contribute/docs/widgets-and-shortcodes/

Reply:
Let me move this to that section.